core/test/computed/metrics/interactive-test.js

```diff
@@ -26,16 +40,20 @@ function generateNetworkRecords(records, timeOrigin) {
       startTime: (item.start + timeOriginInMs) / 1000,
       endTime: item.end === -1 ? -1 : (item.end + timeOriginInMs) / 1000,
     };
+    return /** @type {LH.Artifacts.NetworkRequest} */ (record);
   });
 }
+
 describe('Metrics: TTI', () => {
+  /** @type {{gatherMode: LH.Result.GatherMode}} */
   const gatherContext = {gatherMode: 'navigation'};
 
   it('should compute a simulated value', async () => {
-    const settings = {throttlingMethod: 'simulate'};
+    const settings = /** @type {LH.Config.Settings} */ (
```

Comment:
a test refactor/util I'd love to see is moving tests like this to real config settings. If we don't want to use the full `initializeConfig` (e.g. this var would be `initializeConfig('navigation', undefined, {throttlingMethod: 'simulate'})`, which is kind of awkward), we could switch to an options object